lib/modules/manager/bundler/__snapshots__/extract.spec.ts.snap

```diff
@@ -1606,7 +1609,7 @@ exports[`modules/manager/bundler/extract extractPackageFile() parses rails Gemfi
       },
     },
     {
-      "currentValue": "">= 3.0.5", "< 3.2"",
+      "currentValue": "">= 3.0.5"",
```

Comment:
This seems like a regression?

Reply:
Good catch. I updated the regex and added a dedicated test.